documentation/airport-guides/katl.md

```diff
@@ -0,0 +1,107 @@
+1,193,710 annual operations in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+Located 20 miles southwest of KATL and almost 30 miles from Downtown Atlanta, The A80 TRACON is one of the busiest control facilities in the world. It controls a huge area over north central
+Georgia, as well as portions of South Carolina and Alabama, and it covers over 200 airports, including the world's busiest airport, Hartsfield-Jackson Atlanta Int'l (KATL).
+
+### Airspace
+Due to its sheer size, the A80 TRACON is divided into 4 main sectors: Atlanta, Macon, Columbus and Athens. However, only the Atlanta sector is available in our simulator; It extends 40nm around KATL up to 14,000ft. Class B airspace exists with a ceiling of 12,500ft over the Atlanta metropolitan area.
+
+### Sectors Available in OpenScope
+The A80 TRACON has a huge number of different positions and it is very difficult to find accurate and up-to-date information on the sectorization of the airspace, so the information below may not be 100% accurate.
+```
+ANW - North-West Approach      -----\
+ANE - North-East Approach           |
+ASW - South-West Approach           |
+ASE - South-East Approach           |
+DN  - North Departure               |
+DS  - South Departure               |---> ATL positions
+DE  - East Departure                |
+DW  - West Departure                |
+FN  - Final North (RWYs 8/2         |
+FC  - Final Center (RWYs 9/27)      |
+FS  - Final South (RWY 10/28)  -----/
+SNW - North-West Satellite
+SNE - North-East Satellite
+SSW - South-West Satellite
+SSE - South-East Satellite
+CSG - Columbus Approach
+MCN - Macon Approach
+AHN - Athens Approach
+```
+The airspace available in Openscope is a combination of all ATL positions in the Atlanta sector.
+
+### Airports of Interest
+
+#### KATL (Primary Airport)
+Hartsfield–Jackson Atlanta International  
+881,355 annual operations
+
+Orignally built on an abandoned race track, the Atlanta Speedway, Candler Field was opened in 1926. Already in 1957, the airfield, then Atlanta Municipal Airport, was the busiest airport in the US, with more than two million passengers passing through it, and, between noon and 2 p.m. each day, it became the world's busiest airport. Today, Hartsfield–Jackson is the world's busiest airport by passenger traffic and aircraft operations, handling over 100 million passengers and almsot one million flights yearly. It is the primary hub of Delta Airlines, who operates over 1,000 daily flights, so, to prevent confusion, taxiway delta is referred to as taxiway dixie.
+
+##### Instrument Procedures
+In order to help controllers with the massive number of aircraft that fly in and out of Atlanta every day, procedures are desgined very carefully and revised regularly to minimize the workload. All procedures for jet aircraft are very automated so a controller's main job is to maintain enough separation between aircraft arriving and departing the area.
```

Comment:
I think ‘the extreme masses of traffic that fly in and out’ sounds better instead of ‘the massive number of aircraft that fly in and out’

Subject to opinion though!

Reply:
Agreed :+1: